haskell.nix/flake.nix

```diff
@@ -1,12 +1,16 @@
 {
   nixConfig = {
-    flake-registry = "https://github.com/serokell/flake-registry/raw/zhenya/ops958-hn-nix-template/flake-registry.json";
+    flake-registry = "https://github.com/serokell/flake-registry/raw/master/flake-registry.json";
   };
 
   inputs = {
     flake-compat = {
       flake = false;
     };
+    haskell-nix = {
```

Comment:
Alternatively, we could just drop `hackage` and `stackage` inputs alltogether and just use `nix flake lock --update-input haskell.nix/hackage` (which I _think_ should work)

Reply:
I think it's better to keep them as local inputs so that simple `nix flake update` will work, it's easy to forget about the extra flags

Reply:
Ok